StravaSwift/ModelEnums.swift

```diff
@@ -110,9 +110,11 @@ public enum WorkoutType: Int {
   - Workout
  **/
 public enum ActivityType: String {
-    case ride, run, swim, hike, walk, alpineSki, backcountrySki, canoeing, crossfit, eBikeRide, elliptical, iceSkate,
-    inlineSkate, kayaking, yoga, kitesurf, nordicSki, rockClimbing, rollerSki, rowing, snowboard, snowshoe, stairStepper,
-    standUpPaddling, surfing, virtualRide, weightTraining, windsurf, workout, handcycle, virtualRun, wheelchair
+    case
+    AlpineSki, BackcountrySki, Canoeing, Crossfit, EBikeRide, Elliptical, Golf, Handcycle, Hike, IceSkate, 
+ InlineSkate, Kayaking, Kitesurf, NordicSki, Ride, RockClimbing, RollerSki, Rowing, Run, Sail, Skateboard, 
+ Snowboard, Snowshoe, Soccer, StairStepper, StandUpPaddling, Surfing, Swim, Velomobile, VirtualRide, VirtualRun, 
+ Walk, WeightTraining, Wheelchair, Windsurf, Workout, Yoga
```

Comment:
Activity types should be lowercase.

Reply:
Are you sure - according to the Strava API they have initial capitals, that's why I changed it - see [https://developers.strava.com/docs/reference/#api-models-ActivityType](url)

Reply:
I have put a flow in place to handle initial authorisation and token refresh but it's not completely isolated from my main app code at the moment. I'll take a look and see if it can easily be extracted into a generic process.